week1/pythonWeek01.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# 1. Write a python script to print your name and age
+
+# In[ ]:
+
+
+a = input("Name")
+
+b = int(input("Age"))
+
+print("Your name is : ", a, " Your age is : ", b)
+
+
+# 2. Create a list of your 5 favorite movies and store it in the variable
+
+# In[ ]:
+
+
+favorite_movies = ['Bourne triology','Good will hunting','James bond','mission impossible','the wire']
+
+
+# 3. Write a Python program to display the first and last colors from the following list. 
+# color_list = ["Red","Green","White" ,"Black"]
+
+# In[ ]:
+
+
+color_list = ["Red","Green","White" ,"Black"]
+ 
+print(color_list[0],color_list[-1])
+
+
+# 4. Write a Python script to add a key to a dictionary
+# Sample Dictionary : {0: 10, 1: 20}
+# Expected Result : {0: 10, 1: 20, 2: 30}
+# 
+
+# In[ ]:
+
+
+sample_dictionary = {0:10, 1:20}
+print(sample_dictionary)
+sample_dictionary.update({2:30})
```

Comment:
that's how you merge two dictionaries!
adding one element is done like this:
```python
sample_dictionary[2] = 30
```

Reply:
I think `update` does the same as `dict[key] = value`, although @Rassibassi is the usual way to do it. 

ref: https://www.w3schools.com/python/ref_dictionary_update.asp